package/lib-edgetpu/.cm/meta.json

```diff
@@ -0,0 +1,49 @@
+{
+  "customize": {
+    "extra_dir": "",
+    "install_env": {
+      "PACKAGE_URL": "https://packages.cloud.google.com/apt/pool/",
+      "PACKAGE_WGET": "YES",
+      "PACKAGE_WGET_EXTRA": "--no-cookies"
+    },
+    "version": "with-variations"
+  },
+  "end_full_path": {
+    "linux": "usr/lib/aarch64-linux-gnu/libedgetpu.so.1"
```

Comment:
Perhaps just `libedgetpu.so.1`? It's unique enough, and will allow variations for other architectures too?

Reply:
Ok. Without full path to libedgetpu.so.1 installition works incorrectly.

Reply:
Ok. Without full path to libedgetpu.so.1 installation works incorrectly.